### DISCUSSION TITLE
Flag member projects with 'sponsor me' option available

### DISCUSSION THREAD

**Discussion by sasakrsmanovic** [dotnet-foundation/Home#79] [General Discussions]
Last week I sponsored 23 projects/maintainers in .net ecosystem. More details as to why [here  ](https://platform.uno/blog/mvps-uno-platform-thanks-you/)

The process of finding people/projects to sponsor was very painful. I tried going through the list of projects listed on .net foundation - and that didn't help at all. 

I can imagine more of these cases where a non-technical (marketing, admin) person in an organization is asked to spend $X on sponsoring random projects in .net ecosystem - they'd have very hard time to locate people to sponsor. This process should be made easier. My proposal is to add an 'Open for Sponsorships' button to each project which is either 1-sponsor-able or 2- has maintainers who are open to sponsorships 

![image](https://user-images.githubusercontent.com/33944563/160652906-5b06065f-f95c-49a0-8e9a-4ad3969efedb.png)

**Comment by psmulovics** [comment]
Coming from a corporation angle and wearing that hat, I'm a huge supporter of this idea - many cases a company want to throw money for this cause (PR value? Recruiting vehicle? Etc) and it should be super easy for a non tech people from a marketing department to find the right opportunities.

**Comment by nicoleabuhakmeh** [comment]
Thank you for the suggestion. I'm tagging in board members and the project maintainers group as a way to get them involved on the process.  @rprouse @devlead
